cf-agent/nfs.c

```diff
@@ -496,25 +495,27 @@ int VerifyInFstab(EvalContext *ctx, char *name, const Attributes *a, const Promi
 
     Log(LOG_LEVEL_VERBOSE, "Verifying '%s' in '%s'", mountpt, VFSTAB[VSYSTEMHARDCLASS]);
 
+    int changes = 0;
```

Comment:
I'd prefer moving this near the top of the function and then doing `return changes` in all `return` places.

Reply:
Thanks for feedback, I will keep it in mind for the next time, but don't think it's big enough to do another amend + force push + re-review.